lale/lib/sklearn/dummy_regressor.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2019 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import sklearn
+import sklearn.dummy
+
+import lale.docstrings
+import lale.operators
+
+_hyperparams_schema = {
+    "allOf": [
+        {
+            "description": "This first object lists all constructor arguments with their types, but omits constraints for conditional hyperparameters.",
+            "type": "object",
+            "relevantToOptimizer": [],
+            "additionalProperties": False,
+            "required": ["strategy", "quantile"],
+            "property": {
+                "strategy": {
+                    "description": """Strategy to use to generate predictions.
+- “mean”: always predicts the mean of the training set
+- “median”: always predicts the median of the training set
+- “quantile”: always predicts a specified quantile of the training set, provided with the quantile parameter.
+- “constant”: always predicts a constant value that is provided by the user.""",
+                    "enum": ["mean", "median", "quantile", "constant"],
+                    "default": "mean",
+                },
+                "random_state": {
```

Comment:
the regressor does not have a random state argument.  please delete.  instead, please add
```
"constant": {
                    "description": "The explicit constant as predicted by the “constant” strategy. This parameter is useful only for the “constant” strategy.",
                    "anyOf": [
                        {"type": ["integer", "string"]},
                        {"enum": [None]},
                        {"default": None},
                    ],
                },
```